app/ui-react/packages/ui/src/Shared/HelpDropdown.tsx

```diff
@@ -46,6 +47,61 @@ export class HelpDropdown extends React.Component<
   public render() {
     const { isOpen } = this.state;
     const dropdownItems = [
+      <DropdownItem
+        key="sampleIntegrationTutorials"
+        component="span"
+        onClick={() => {
+          window.open(
+            'https://access.redhat.com/documentation/en-us/red_hat_fuse/7.3/html-single/fuse_online_sample_integration_tutorials/',
```

Comment:
I think all these click handlers should be exposed to the app. These links shouldn’t be hard coded, not in the ui anyway.